phpBB/phpbb/event/kernel_terminate_subscriber.php

```diff
@@ -35,7 +35,7 @@ public function on_kernel_terminate(PostResponseEvent $event)
 	public static function getSubscribedEvents()
 	{
 		return array(
-			KernelEvents::TERMINATE		=> 'on_kernel_terminate',
+			KernelEvents::TERMINATE		=> array('on_kernel_terminate', -10000),
```

Comment:
about the tests, you mean for `array('on_kernel_terminate', -10000)` ? It's part of the SYmfony api.